kernel/env.ml

```diff
@@ -52,7 +52,7 @@ let _define (l:loc) (id:ident) (te:term) (ty_opt:typ option) : unit =
   match ty with
   | Kind -> raise (DefineExn (l,id))
   | _ ->
-    _declare l id Signature.Definable ty;
+    _declare l id Signature.Injective ty;
```

Comment:
Why should a symbol declare by `:=` be injective?
I think we should let `Signature.Definable` here.

Reply:
Should be fixed now.